cost/aws/old_snapshots/README.md

```diff
@@ -0,0 +1,64 @@
+# AWS Old Snapshots
+
+## What it does
+
+This Policy finds AWS snapshots older than the specified days and deletes them.
```

Comment:
Should also include details about filtering on the owner id and how you are getting the owner id.